.github/workflows/main.yml

```
@@ -0,0 +1,13 @@
+name: CI
+
+on: [push]
```

Maybe like this? [link from github community](https://github.community/t5/GitHub-Actions/How-to-trigger-an-action-on-push-or-pull-request-but-not-both/td-p/35805)
```suggestion
on:
  push:
    branches:
    - master
  pull_request:
    branches:
    - master
```

---

It also can be 
```
on: 
  pull_request:
  push:
    branches:
      - master
```

---

А зачем, если нужно всегда?

---

@humb1t  потому что как я понимаю on push  - это ты запушил в любую ветку коммит, пуллреквесты не считаются. Можно проверить конечно, я особо по доке не понял
можно тогда так :
`on: [push, pull_request]`
[push](https://help.github.com/en/actions/reference/events-that-trigger-workflows#push-event-push)
[pull_request](https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-event-pull_request)

---

@kortov - закинешь PR?

---

Ок, позжо